README.rst

```diff
@@ -185,9 +185,9 @@ with any future changes. For example:
 
     from corsheaders.defaults import default_methods
 
-    CORS_ALLOW_METHODS = default_methods + (
```

Comment:
OOoh, this code snippet won't work - `default_methods` is a tuple (deliberately, so library defaults are immutable) so this will fail with:

```
TypeError: can only concatenate tuple (not "list") to tuple
```

Probably `list(default_methods) + [` will do

Reply:
Yes, I agree that's the best way to do it. Using a list gives the user the ability to extend the option in another config (e.g. another environment) using either a list or a tuple:

`CORS_ALLOW_METHODS += ["foo"]`
`CORS_ALLOW_METHODS += ("bar",)`